### PR TITLE
PR to discuss about zoom value on status bar

### PR DESCRIPTION
**Pull request by SinghRajenM**
Hi @donho and All,

I am trying to implement a feature to show zoom value on status bar which was discussed on community and issue tracker.
https://notepad-plus-plus.org/community/topic/15679/seeing-the-zoom-level-with-pythonscript
https://github.com/notepad-plus-plus/notepad-plus-plus/issues/4566
https://notepad-plus-plus.org/community/topic/17321/feature-request-current-zoom-to-status-bar

I was able to reach half way. Current solution is partially working as it works on click on zoom in/out from toolbar button and menu item (View->Zoom->options). It is not working for shortcuts e.g. <kbd>Ctrl</kbd>+<kbd>+</kbd>, or <kbd>Ctrl</kbd>+ <kbd>mouse scroll</kbd> etc.
But now I'm stuck. I'm not able to figure out how scintilla shortcut works. Like how scintilla accelerator works, which function is called or which event/message is sent? A help to understand this would be appreciated. 

image

### CODE REVIEW

**Comment by sasumner** [comment]
IMO Zoom is not important enough to waste valuable status bar space on. :)

**Comment by Ekopalypse** [comment]
Scintilla shortcut is CTRL and +(plus sign from numpad) and notification is [SCN_ZOOM](https://www.scintilla.org/ScintillaDoc.html#SCN_ZOOM)

File: PowerEditor/src/Notepad_plus.cpp [hunk 1]
```diff
@@ -2346,6 +2351,28 @@ void Notepad_plus::setLangStatus(LangType langType)
 	_statusBar.setText(getLangDesc(langType).c_str(), STATUSBAR_DOC_TYPE);
 }
 
+void Notepad_plus::setPercentStatus(int whichOne, int percent)
```
Comment (MAPJe71) [inline PowerEditor/src/Notepad_plus.cpp]: Renaming parameter `whichOne` to `whichView` would make its purpose more clear IMO.

**Comment by Ekopalypse** [comment]
@SinghRajenM 
I don't want to hijack this PR with another request but as it belongs to statusbar and there is already
a request open #1268 I wanted to take the chance to ask if this could be implemented as well.
From my point of view, when working with scintilla this one would be quite nice to have.
I know, you can use goto window to see where am I but I think it would be even nicer to have it,
maybe after len:... line:..., visible in the statusbar.
If you disagree with it - don't mind and ignore this request.
Thank you

File: PowerEditor/src/Notepad_plus.cpp (hunk 1 above)
Comment (donho) [reply PowerEditor/src/Notepad_plus.cpp]: What's your suggestion @MAPJe71 ?

**Comment by donho** [comment]
@SinghRajenM 
I second what @sasumner said.

**Comment by SinghRajenM** [comment]
Cool!!!

@donho but there should be way to know current level, isn't it?

**Comment by donho** [comment]
@SinghRajenM  It could be "nice to have", but it's not a mandatory info IMO. 
This info could be on both tooltip of zoom - if the implementation is trivial. Otherwise let's focus our coding effort on more important feature and critical bug.

**Comment by n3tcom** [comment]
May I ask, on the record, is it the official position of Notepad++ that this critical accessibility feature is not of importance?

**Comment by sasumner** [comment]
>May I ask, on the record, is it the official position of Notepad++ that this critical accessibility feature is not of importance?

Official position, on the record:  Not important.

**Comment by kipusoep** [comment]
What is wrong with giving people the power to decide for themselves if this is valuable or not?
Implement the feature and make it optional (opt-in) would make both sides happy I'd say...

**Pull request closed without merging**